lang/src/main/antlr4/org/kaazing/k3po/lang/parser/v2/Robot.g4

```diff
@@ -41,7 +41,11 @@ acceptableNode
     ;
 
 connectNode
-    : k=ConnectKeyword (AwaitKeyword barrier=Name ConnectKeyword)? connectURI=location (OptionKeyword TransportKeyword value=location)? streamableNode+
+    : k=ConnectKeyword (AwaitKeyword barrier=Name ConnectKeyword)? connectURI=location
+                       (OptionKeyword TransportKeyword value=location)?
```

Comment:
We also don't want the options to be repeated. If we can write the grammar, that would be good.